packages/target-tools/src/telemetryProvider.spec.js

```diff
@@ -72,15 +77,16 @@ describe("TelemetryProvider", () => {
             prefetchMboxCount: expect.any(Number),
             prefetchViewCount: expect.any(Number)
           },
-          execution: 1,
-          parsing: 2,
+          execution: 1.11,
+          parsing: 2.22,
           request: {
             dns: expect.any(Number),
             tls: expect.any(Number),
             timeToFirstByte: expect.any(Number),
             download: expect.any(Number),
             responseSize: expect.any(Number)
```

Comment:
@dcottingham We'd probably want to reduce the double precision for these fields, too.